docs/app-host/configuration.md

```diff
@@ -0,0 +1,89 @@
+
+## Resource service
+
+| Option | Default Value | Description |
+|--|--|--|
+| `DOTNET_RESOURCE_SERVICE_ENDPOINT_URL` | Automatically generated 128-bit entropy token. | Configures the address of the resource service hosted by the app host. Automatically generated with `launchSettings.json` to have a random port on localhost. For example, `https://localhost:17037`. |
+| `DOTNET_DASHBOARD_RESOURCESERVICE_APIKEY` | `null` | The API key used to authenticate requests made to the app host's resource service. The API key is required if the app host is in run mode, the dashboard isn't disabled, and the dashboard isn't configured to allow anonymous access with `DOTNET_DASHBOARD_UNSECURED_ALLOW_ANONYMOUS`. |
+
+## Dashboard
+
+By default, the dashboard is automatically started by the app host. The dashboard supports [its own set of configuration](../fundamentals/dashboard/configuration.md), and some settings can be configured from the app host.
+
+| Option | Default Value | Description |
+|--|--|--|
+| `ASPNETCORE_URLS` | `null` | Dashboard address. Must be https unless `ASPIRE_ALLOW_UNSECURED_TRANSPORT` or `DistributedApplicationOptions.AllowUnsecuredTransport` is true. Automatically generated with `launchSettings.json` to have a random port on localhost. The value in launch settings is set on the `applicationUrls` property. |
+| `ASPNETCORE_ENVIRONMENT` | `Production` | Configures the environment the dashboard runs as. For more information, see [Use multiple environments in ASP.NET Core](https://learn.microsoft.com/aspnet/core/fundamentals/environments). |
+| `DOTNET_DASHBOARD_OTLP_ENDPOINT_URL` | `http://localhost:18889` if no gRPC endpoint is configured. | Configures the dashboard OTLP gRPC address. Used by the dashboard to receive telemetry over OTLP. Set on resources as the `OTEL_EXPORTER_OTLP_ENDPOINT` env var. The `OTEL_EXPORTER_OTLP_PROTOCOL` env var is `grpc`.  Automatically generated with `launchSettings.json` to have a random port on localhost. |
+| `DOTNET_DASHBOARD_OTLP_HTTP_ENDPOINT_URL` | `null` | Configures the dashboard OTLP HTTP address. Used by the dashboard to receive telemetry over OTLP. If only `DOTNET_DASHBOARD_OTLP_HTTP_ENDPOINT_URL` is configured then it is set on resources as the `OTEL_EXPORTER_OTLP_ENDPOINT` env var. The `OTEL_EXPORTER_OTLP_PROTOCOL` env var is `http/protobuf`. |
+| `DOTNET_DASHBOARD_FRONTEND_BROWSERTOKEN` | Automatically generated 128-bit entropy token. | Configures the frontend browser token. This is the value that must be entered to access the dashboard when the auth mode is BrowserToken. If no browser token is specified then a new token is generated each time the app host is launched. |
+
+## Internal
+
+Internal settings are used by the app host and integrations. Internal settings aren't designed to be configured directly.
+
+| Option | Default Value | Description |
+|--|--|--|
+| `AppHost:Directory` | The content root if there's no project. | Directory of the project where the app host is located. |
```

Comment:
Is this exposed through the distributed app builder as the [`IDistributedApplicationBuider.AppHostDirectory`](https://learn.microsoft.com/en-us/dotnet/api/aspire.hosting.idistributedapplicationbuilder.apphostdirectory?view=dotnet-aspire-9.0#aspire-hosting-idistributedapplicationbuilder-apphostdirectory)? Might be good to add an xref to that.
```suggestion
| `AppHost:Directory` | The content root if there's no project. | Directory of the project where the app host is located. Accessible from the <xref:Aspire.Hosting.IDistributedApplicationBuilder.AppHostDirectory?displayProperty=nameWithType>. |
```